css/style.css

```diff
@@ -8,7 +8,40 @@
   padding: 0;
 }
 
+body {
+  overflow-x: hidden;
+}
+
```

Comment:
- [ ] The same issue that previous reviewers mentioned is happening again, kindly check the video below.  I would suggest you check the min-width and max-width that you have given in your project and go from there https://www.loom.com/share/31816149330d4ca889600c05343c24a6